src/layout/Footer/Footer.js

```diff
@@ -23,6 +23,26 @@ export default function Footer() {
           <Nav.Link target="_blank" href="https://github.com/ogroppo/entitree">
             <FaGithub />
           </Nav.Link>
+          {/*<Nav.Link*/}
+          {/*  target="_blank"*/}
+          {/*  href="https://twitter.com/EntiTreeApp?ref_src=twsrc%5Etfw"*/}
+          {/*  style={{*/}
+          {/*    color: "blue",*/}
+          {/*  }}*/}
+          {/*>*/}
+          {/*  <FaTwitter />*/}
+          {/*</Nav.Link>*/}
+          <Nav.Item id="twitterFollowButton">
+            <a
+              href={`https://twitter.com/EntitreeApp?ref_src=twsrc%5Etfw`}
+              className="twitter-follow-button"
+              // data-size="large"
+              // data-show-screen-name=""
+              // data-show-count=""
+            >
+              Follow EntiTree
+            </a>
+          </Nav.Item>
```

Comment:
Hopefully the \<TwitterTimelineEmbed/\> will work for you

Reply:
what's that?